core/client/src/main/java/alluxio/client/AbstractOutStream.java

```diff
@@ -40,7 +40,7 @@ public int getBytesWritten() {
   /**
    * Aborts the output stream.
    *
-   * @throws IOException if there is a failure to abort this output stream
+   * @throws IOException if an I/O error occurs
```

Comment:
I'd prefer just `@throws IOException` without anything extra (this is what HDFS does), but I couldn't find a way to tell checkstyle that `IOException` and `AlluxioException` don't require javadoc descriptions. The alternative is to relax the check for all `@throws` clauses. I'd be OK with that, what do you think?